charts/catalog-api/templates/secret-deploys.yaml

```
@@ -9,5 +9,26 @@ metadata:
     {{- end }}
 type: Opaque
 data:
-  apiDbPassword:  {{ required "A valid .Values.api.postgres.password entry required" .Values.api.postgres.password | b64enc }}
-  keysServiceToken:  {{ required "A valid .Values.keys.token entry required" .Values.keys.token | b64enc }}
+  {{- if .Values.api.postgres.tls.enabled }}
+  apiDbPassword: {{ .Values.api.postgres.password | b64enc }}
```

На пустой пароль я получаю тут ошибку, т.к. применить на пустую строку `| b64enc` нельзя.
Давай сразу пустое значение сделаем. Т.к. задание пароля отменяет авторизацию по mTLS.
```suggestion
  apiDbPassword: ''
```
Здесь и далее по коду, и для `importer.postgres.password` тоже надо поправить.